src/V3ParseGrammar.cpp

```diff
@@ -84,9 +84,14 @@ AstNode* V3ParseGrammar::argWrapList(AstNode* nodep) {
 }
 
 AstNode* V3ParseGrammar::createSupplyExpr(FileLine* fileline, const string& name, int value) {
-    return new AstAssignW(
-        fileline, new AstVarRef(fileline, name, VAccess::WRITE),
-        new AstConst(fileline, AstConst::StringToParse(), (value ? "'1" : "'0")));
+    AstAssignW* assignp
+        = new AstAssignW(fileline, new AstVarRef(fileline, name, VAccess::WRITE),
+                         new AstConst(fileline, AstConst::StringToParse(), (value ? "'1" : "'0")));
+    AstStrengthSpec* strengthSpecp
+        = new AstStrengthSpec(fileline, new AstStrength(fileline, AstStrength::SUPPLY, 0),
+                              new AstStrength(fileline, AstStrength::SUPPLY, 1));
```

Comment:
```suggestion
        = new AstAssignW{fileline, new AstVarRef{fileline, name, VAccess::WRITE},
                         new AstConst{fileline, AstConst::StringToParse{}, (value ? "'1" : "'0")}};
    AstStrengthSpec* strengthSpecp
        = new AstStrengthSpec{fileline, new AstStrength{fileline, AstStrength::SUPPLY, 0},
                              new AstStrength{fileline, AstStrength::SUPPLY, 1}};
```
This is the new Verilator style. They're going to request that, so you may as well do it now. Do it in the rest of the patch as well.